OpenRA.Mods.Cnc/Widgets/ModelWidget.cs

```diff
@@ -190,9 +190,6 @@ public override void PrepareRenderables()
 			if (cameraAngle != cachedCameraAngle)
 				cachedCameraAngle = cameraAngle;
 
-			if (cachedVoxel == null)
-				return;
-
```

Comment:
You remove `cachedVoxel` in the 4th commit. Why is this separated to the first commit?

Reply:
I can merge both together if you want, this was just the first change I made on this file, see the commit description. (This condition was always false.)